client/components/cocktail-list.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+class Cocktails extends React.Component {
+
+  constructor (props) {
+    super(props);
+    this.state = {
+      list: props.cocktails
```

Comment:
The eventual goal will be to filter the list as the user types ingredients, so I think you're right to store cocktails in state :+1: (which can change, while props can't). Let's make this
```
this.state = {
  cocktails: props.cocktails
}l
```
... and then map over `this.state.cocktails` when rendering.